### ISSUE TITLE
Cannot access the app rating score radio buttons using the keyboard

### ISSUE THREAD

**Issue by ctrance** [ucla-oarc-mobile/casa-on-rails#214]
@krozett Cannot access the app rating score radio buttons using the keyboard
![image](https://user-images.githubusercontent.com/1034174/108132074-2040eb80-7067-11eb-8986-c0f1aa9de9d5.png)

Accessibility success criterion:
* https://www.w3.org/TR/WCAG21/#keyboard

**Comment by ctrance** [comment]
Closing old tickets that we will not fix for now.
